### PR TITLE
Project 2(elderly wearable webpage)

### PR DESCRIPTION
**Pull request by aaryanbansal**

### CODE REVIEW

**Review by doubleedesign — CHANGES_REQUESTED** [review]
- Please refactor to work with React.
- All code needs to be in the existing React app in the `src` folder. Please do not add a new React app or any other new folders other than those required to create your routes and components.

Please see the [guides](https://redback-operations.github.io/redback-documentation/docs/category/front-end-web-development) on the company documentation website for more information.

File: src/html/ExploreFeature.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import './styles.css'; // Import your styles.css file
+import healthMonitoringImage from './healthmonitoring.jpeg'; // Import images
+import medicationReminderImage from './medication reminder.jpeg';
+
+class ExploreFeatures extends React.Component {
```
Comment (doubleedesign) [inline src/html/ExploreFeature.js:6]: Please use function components, as class components are deprecated. [Here's some more information](https://react.dev/reference/react/Component) including how to migrate.

File: src/html/contact.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```
Comment (doubleedesign) [inline src/html/contact.html]: Please refactor all HTML in to React components. There should not be any new `.html` files.

File: src/html/common.css [hunk 3]
```diff
@@ -0,0 +1,333 @@
+/* External CSS style file */
```
Comment (doubleedesign) [inline src/html/common.css:1]: Please refactor all CSS into Styled Components. You can find some more information here: https://redback-operations.github.io/redback-documentation/docs/web-mobile-app-dev/frontend/styled-components

**Review by doubleedesign — CHANGES_REQUESTED** [review]
Please update to follow current React standards and the existing file structure and naming conventions in the app. You can find step-by-step instructions on how to do much of this in the company documentation website. In particular:
- [Pages and Routes](https://redback-operations.github.io/redback-documentation/docs/web-mobile-app-dev/frontend/new-routes)
- [Creating components](https://redback-operations.github.io/redback-documentation/docs/web-mobile-app-dev/frontend/new-components)
- [Using Styled Components](https://redback-operations.github.io/redback-documentation/docs/web-mobile-app-dev/frontend/styled-components)

**Comment by ben-AI-cybersec** [comment]
Tested locally. None of this code appears to work with the react app. Closing PR

**Pull request closed without merging**